src/loaders/ObjectLoader.js

```diff
@@ -26,11 +28,17 @@ import { MaterialLoader } from './MaterialLoader';
 import { BufferGeometryLoader } from './BufferGeometryLoader';
 import { JSONLoader } from './JSONLoader';
 import { XHRLoader } from './XHRLoader';
+import * as CoreGeometry from '../core/Geometries';
+import * as ExtraGeometry from '../extras/geometries/Geometries';
 
 /**
  * @author mrdoob / http://mrdoob.com/
  */
 
+function resolveGeometryType ( type ) {
```

Comment:
@mrdoob I'm so sorry! I've been flat-out with work recently and being in the process of buying a house has sucked up most of my free time. Thanks, for taking over, much appreciated!